funcx_endpoint/tests/funcx_endpoint/executors/high_throughput/test_funcx_manager.py

```diff
@@ -17,7 +17,12 @@ def test_setup_teardown(self):
         yield
         shutil.rmtree(os.path.join(os.getcwd(), 'mock_uid'))
 
-    def test_remove_worker_init(self):
+    def test_remove_worker_init(self, mocker):
+        mocker.patch('funcx_endpoint.executors.high_throughput.funcx_manager.zmq.Context')
+
+        mock_thread = mocker.patch('funcx_endpoint.executors.high_throughput.funcx_manager.threading.Thread')
```

Comment:
Let's confirm that the tests actually need this to be mocked. If not, I'd skip mocking this.